python/ray/data/tests/test_executor_resource_management.py

```diff
@@ -292,7 +292,7 @@ def test_actor_pool_resource_reporting(ray_start_10_cpus_shared, restore_data_co
     assert op.incremental_resource_usage() == ExecutionResources(
         cpu=0, gpu=0, object_store_memory=inc_obj_store_mem
     )
-    assert op.current_processor_usage() == ExecutionResources(cpu=2, gpu=0)
+    assert op.current_processor_usage() == ExecutionResources(cpu=0, gpu=0)
```

Comment:
sorry, we shouldn't update this. See my comment in `current_processor_usage`.

Reply:
No worries, will revert these.